user-interface/src/main/java/life/qbic/datamanager/views/register/UserRegistrationComponent.java

```diff
@@ -35,7 +35,7 @@
  */
 @SpringComponent
 @UIScope
-public class UserRegistrationComponent extends Div {
+public class UserRegistrationComponent extends CardLayout {
```

Comment:
Why again the card layout?

Reply:
The Cardlayout is a DIV Container with a specific style. This style is the same for all components related to password reset, user registration and login. Therefore it's easier to maintain if we have one java class which defines the style and reuse it in the different main classes